<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listworks</class>
 <widget class="QMainWindow" name="listworks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>-50</y>
      <width>671</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(1, 59, 149);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>321</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 18pt &quot;Dast Nevis&quot;;
color: rgb(1, 59, 149);</string>
    </property>
    <property name="text">
     <string>     Todolist</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/لوگو.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>    usrname</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/tasks.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/tasks.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/tasks.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/tasks.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_list1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(1, 59, 149);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>.....</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_list2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(1, 59, 149);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>.....</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_list3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(1, 59, 149);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>.....</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_list4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>580</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(1, 59, 149);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>.....</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>       Creat a new list</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>661</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_color">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>choose color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_font">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>choose font</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_listname">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>173</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Oswald&quot;;
color: rgb(61, 61, 61);</string>
    </property>
    <property name="placeholderText">
     <string>list name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/profile.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>661</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addlist">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/tasks.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_starlist">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>star work list</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>960</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/tasks.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>970</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>star work list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_assigned">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>470</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>List of assigned tasks</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/tasks.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>560</y>
      <width>661</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>570</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> assigne work to your friend :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>580</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Dast Nevis&quot;;
color: rgb(61, 61, 61);</string>
    </property>
    <property name="placeholderText">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_workname">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>620</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Dast Nevis&quot;;
color: rgb(61, 61, 61);</string>
    </property>
    <property name="placeholderText">
     <string>work name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sendtask">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>650</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>send task </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_eror">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton_list1</zorder>
   <zorder>pushButton_list2</zorder>
   <zorder>pushButton_list3</zorder>
   <zorder>pushButton_list4</zorder>
   <zorder>label_10</zorder>
   <zorder>line</zorder>
   <zorder>pushButton_color</zorder>
   <zorder>pushButton_font</zorder>
   <zorder>lineEdit_listname</zorder>
   <zorder>label_11</zorder>
   <zorder>line_2</zorder>
   <zorder>pushButton_addlist</zorder>
   <zorder>label_12</zorder>
   <zorder>pushButton_starlist</zorder>
   <zorder>label_13</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_assigned</zorder>
   <zorder>label_15</zorder>
   <zorder>line_3</zorder>
   <zorder>label_14</zorder>
   <zorder>lineEdit_username</zorder>
   <zorder>lineEdit_workname</zorder>
   <zorder>pushButton_sendtask</zorder>
   <zorder>label_eror</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
